docs/DebuggingTheCompiler.md

```diff
@@ -807,6 +812,74 @@ reducing SIL test cases by:
 For more information and a high level example, see:
 ./swift/utils/bug_reducer/README.md.
 
+## Debugging the Compiler Driver
+
+The Swift compiler uses a standalone compiler-driver application:
```

Comment:
I would make it clear here that the application is written in swift.

Reply:
Done.